bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/IFindReplaceLogic.java

```diff
@@ -66,6 +66,14 @@ public interface IFindReplaceLogic {
 	 */
 	public boolean isRegExSearchAvailableAndActive();
 
+	/**
+	 * Returns whether the Find/Replace-Overlay may perform incremental search
+	 * currently.
+	 *
+	 * @return whether incremental search may be performed with the current options.
```

Comment:
This is the logic interface, which is completely independent from UI / overlay. If a method is only a getter (and thus only needs a "returns" documentation), you can also use a simplified Javadoc syntax: `{@return ... }
```suggestion
	 * @return whether incremental search may be performed based on the currently active options
```

Reply:
I am not quite sure what you mean with that syntax?
I did adapt your JavaDoc-text change

Reply:
The javadoc still refers to UI. But unfortunately my proposal was incomplete, sorry for that. This is what I would propose to have the javadoc look like:

```java
/**
  * {@return whether incremental search may be performed based on the currently active options}
  */
```

Then it renders as follows:
<img width="544" alt="image" src="https://github.com/eclipse-platform/eclipse.platform.ui/assets/755472/f0c25435-84fb-471a-8f5b-6ce98b096976">

Reply:
I didn't know this was possible. Thank you for the suggestion!